fallout/turfs/plating.dm

```diff
@@ -257,32 +257,51 @@
 /turf/open/floor/plating/ground/road
 	name = "\proper road"
 	desc = "A stretch of road."
-	icon = 'fallout/icons/turf/roadsidewalk.dmi'
-	icon_state = "road"
-	var/dir_variation = TRUE
+	icon = 'fallout/icons/turf/road.dmi'
+	icon_state = "road-255"
+	base_icon_state = "road"
+	smoothing_flags = SMOOTH_BITMASK
+	smoothing_groups = list(SMOOTH_GROUP_FALLOUT_ROAD)
+	canSmoothWith = list(SMOOTH_GROUP_FALLOUT_ROAD)
 
 /turf/open/floor/plating/ground/road/Initialize()
 	. = ..()
-	if(dir_variation)
-		dir = pick(GLOB.cardinals)
+	addtimer(CALLBACK(src, /atom/.proc/update_icon), 1)
 
-/turf/open/floor/plating/ground/road/curb
-	icon_state = "curb"
-	dir_variation = FALSE
+/turf/open/floor/plating/ground/road/update_icon()
+	var/crack_randomiser = "crack_[rand(1,18)]"
+	var/road_randomiser = "rand(-10,10)"
+	var/direction_randomiser = "rand(0,8)"
+	if(prob(20))
+		add_overlay(image('fallout/icons/turf/road.dmi', crack_randomiser, FLOAT_LAYER, direction_randomiser, road_randomiser, road_randomiser))
+		return
 
-/turf/open/floor/plating/ground/road/curb/corner
-	icon_state = "curbcorner"
-	dir_variation = FALSE
-
-/turf/open/floor/plating/ground/road/sidewalk
+/turf/open/floor/plating/ground/sidewalk
 	name = "sidewalk"
 	desc = "Paved tiles specifically designed for walking upon."
-	icon_state = "sidewalk"
-	dir_variation = FALSE
+	icon = 'fallout/icons/turf/sidewalk.dmi'
+	icon_state = "sidewalk-255"
+	base_icon_state = "sidewalk"
+	smoothing_flags = SMOOTH_BITMASK
+	smoothing_groups = list(SMOOTH_GROUP_FALLOUT_SIDEWALK)
+	canSmoothWith = list(SMOOTH_GROUP_FALLOUT_SIDEWALK, SMOOTH_GROUP_FALLOUT_WALL, SMOOTH_GROUP_FALLOUT_WALL_METAL, SMOOTH_GROUP_FALLOUT_WALL_WOOD, SMOOTH_GROUP_FALLOUT_WALL_SCRAP, SMOOTH_GROUP_FALLOUT_WALL_ADOBE, SMOOTH_GROUP_FALLOUT_WALL_BRICK, SMOOTH_GROUP_FALLOUT_WALL_REINFORCED, SMOOTH_GROUP_FALLOUT_WINDOW)
+
+/turf/open/floor/plating/ground/sidewalk/Initialize()
+	. = ..()
+	addtimer(CALLBACK(src, /atom/.proc/update_icon), 1)
```

Comment:
Why does this require a timer? Do you have a different result if you do it without one?

Reply:
Yes, the whole reason it uses a one use timer is to update the Icon Post Initialisation, because of smoothing code overriding that entire start proc, this just sets it so a different proc is guaranteed to be triggered once after Initialisation.

Reply:
From what I read you want to guarantee `icon_state` is changed and an overlay is added.
Smoothing code will override the `icon_state` change, so the timer will only help initially, and then be overriding as soon a s a neighboring smoothing icon changes.
This shouldn't smooth.

Reply:
no, nothing to do with icon_state changes, only overlays.